Client/Frontend/Sync/SyncSettingsTableViewController.swift

```diff
@@ -130,6 +135,16 @@ class SyncSettingsTableViewController: UITableViewController {
         present(actionShet, animated: true)
     }
     
+    private func showNoConnectionAlert() {
```

Comment:
This is directly copied from `SyncViewController` let's abstract this function and have it return a `UIAlertController`. Best not to copy.